### ISSUE TITLE
link the new web site to http://openmath.org

### ISSUE THREAD

**Issue by kohlhase** [OpenMath/OpenMath.github.io#30]
The OM BM decided to make the new web site at openmath.github.io the official one. 
This now needs to be put into effect. There are some things to do 
- [ ] announce that the old web site is obsolete and maybe redirect to 
- [ ] change the DNS (James can do this I think) 
- [ ] do the CNAME magic on github, 
- [ ] I am sure there must be more to this 
We may have to set up some travis stuff to run the indexer automatically. 
I am assigning to all people who may be involved with some steps of this.

**Comment by davidcarlisle** [comment]
Mostly this has to be done at the DNS provider end see the docs at


https://help.github.com/articles/using-a-custom-domain-with-github-pages/

I can (probbaly) fiddle with the github settings if I have DNS information

**Comment by tkw1536** [comment]
Configuring the hosting should be rather simple. @davidcarlisle is right, most of it has to be done with the DNS provider. 
> change the DNS
  1. Setup two `A` records for `openmath.org`, pointing to `192.30.252.153` and `192.30.252.154` respectively. 
  2. (Optional, but nice) Setup a `CNAME` record for `www.openmath.org` to point to `openmath.org`

>  do the CNAME magic on github

This part is simple, just do `echo "openmath.org" > CNAME` and commit & push. 


>  I am sure there must be more to this

The one thing I can think of is the mailman web interface currently configured to run under [http://openmath.org/mailman/admin/om3](http://openmath.org/mailman/admin/om3). This should probably be moved (maybe to a separate domain like mailman.openmath.org). @jwknopper Do you know the setup?

>  We may have to set up some travis stuff to run the indexer automatically.

Do we need an indexer? If so how is it currently run (is there documentation)? There also is [jekyll-sitemap](https://github.com/jekyll/jekyll-sitemap) which generates `sitemap.xml`, that might be sufficient.

**Comment by jwknopper** [comment]
using mailman.openmath.org would be a better option than the current setup (and HTTPS/SSL would be nice; but not critical, since I use an SSH tunnel myself); As soon as mailman.openmath.org exists I can set that up.

at the moment openmath.org keeps /mailman /pipermail /icons /images/mailman (should keep /cgi-bin as well) and sends the rest on to www.openmath.org. For HTTPS the current situation is a bit ugly since I am still using strict host checking with a certificate for a different host so am also keeping /svn/repos for mathdox.org.

**Comment by kohlhase** [comment]
So what is the status of this, can we just do the switchover now? The new site is much much better (more correct, more complete) than the old one.

**Comment by kohlhase** [comment]
At least we need to add a note to the old page that points to the new one! 
@jwknopper could you please do that directly.

**Comment by jwknopper** [comment]
yes, this can be done by changing the frontpage at https://github.com/OpenMath/OMSVN/blob/master/www/frontpage.xml

I'll see if I can do this tonight. Feel free to do this earlier.

Who is the person to create mailman.openmath.org and let it point to 131.155.0.177, possibly with ssl ?
I can then change the mailman config and when that is done, the openmath website can be moved completely.

**Comment by kohlhase** [comment]
> yes, this can be done by changing the frontpage at https://github.com/OpenMath/OMSVN/blob/master/www/frontpage.xml
> 
> I'll see if I can do this tonight. Feel free to do this earlier.

Great, please do. 
> Who is the person to create mailman.openmath.org and let it point to 131.155.0.177, possibly with ssl ?

@JamesHDavenport I think.

> I can then change the mailman config and when that is done, the openmath website can be moved completely.
> 
> Note: current IP is 131.155.0.167, but that machine will be retired soon.

What happens then? Do you have a follow-up machine?

**Comment by jwknopper** [comment]
131.155.0.177 is our newest server, I'll proxy to the original machine and later to the machine (or hopefully container) which will have the migrated lists.

**Comment by kohlhase** [comment]
OK.

**Comment by jwknopper** [comment]
A warning is now in place at https://openmath.org (frontpage). Also the news refers to the new news location.

**Comment by JamesHDavenport** [comment]
Excellent – many thanks.
James

**Comment by JamesHDavenport** [comment]
Um – not sure. I can look into it, though.

**Comment by kohlhase** [comment]
@JamesHDavenport we should consider giving Jan Willem or Tom admin access to the DNS.

**Comment by JamesHDavenport** [comment]
OK- I've looked into this,and it SEEMS as if Mika's company (spiritdomain?) does allthe handling for openmath,org, which SEEMS to be http://www.openmath.org? and an MX record pointing all mail at mx1.tue.nl and mx2.tue.nl - changingthis (and I need to understandbetter what to) would require getting back in touch with spritidomains. I can do this, but need a wet towel round my head first.

What exactly is the question?


James Davenport
Fulbright CyberSecurity Scholar (at New York University)
Hebron & Medlock Professor of Information Technology, University of Bath
National Teaching Fellow 2014
OpenMath Content Dictionary Editor
Director of Studies EPSRC Doctoral Taught Course Centre for HPC
Chair, IMU Committee on Electronic Information and Communication
Vice-President and Academy Trustee, British Computer Society
________________________________

**Comment by kohlhase** [comment]
James, 
if doing the admin at spirit domains is a hassle, we should consider moving the domain to joker.com (I do my domains there) where I know what to do.

**Comment by kohlhase** [comment]
That may be simpler in the end than forcing you to understand all the peculiarities.

**Comment by jwknopper** [comment]
As far as I know the DNS is at cloudflare. All access to the website is filtered and buffered through there. The SSL certificate we have is also a cloudflare certificate. I think Paul set this up in 2013.

openmath.org mail is handled by 10 mx2.tue.nl.
openmath.org mail is handled by 10 mx1.tue.nl.
openmath.org has IPv6 address 2400:cb00:2048:1::681c:a54
openmath.org has IPv6 address 2400:cb00:2048:1::681c:b54
openmath.org has address 104.28.11.84
openmath.org has address 104.28.10.84
openmath.org name server hank.ns.cloudflare.com.
openmath.org name server ruth.ns.cloudflare.com.

Notes:
- www.openmath.org has same addresses as openmath.org
- mail by mx1.tue.nl and mx2.tue.nl makes sense, because most mail servers at TU/e are not allowed direct access to the web. Also the TU/e servers provide a buffer this way.
- SSL certificate is from Cloudflare and protects the connection till cloudflare. I have  a self-signed certificate at the site running the website (but currently not for openmath.org or www.openmath.org).

**Comment by JamesHDavenport** [comment]
Cloudflare is actually what spiritdomains uses. So the question is whether this is actually one use of cloudflare or two. I need to log back into spiritdomains and check.

I agree with Michael that it might be easier to move to a registrar we know better. I believe I have tweaked our configuration at spiritdomains so that this is possible, and we might do it, either now if necessary, or when the next renewal comes up (Feb?).

**Comment by kohlhase** [comment]
I think if we want to transfer

> and we might do it, either now if necessary, or when the next renewal comes up (Feb?).

I think doing the transfer while we do not have to worry about renewal is probably best, the only downside is that we may have to pay double. But I guess it might streamline the web site switching process. 

@JamesHDavenport You can find what is involved at https://joker.com/faq/content/7/69/en/how-to-transfer-a-domain-to-jokercom.html
Basically, James has to "unlock the domain" (sometimes) and request an AUTH-ID. Then I can do the rest.

**Comment by kohlhase** [comment]
Should we try the transfer soon? After mid-October I will be busy with the new semester.

**Comment by tkw1536** [comment]
This has been done ages ago, closing.
